docs/administration/parameters.md

```diff
@@ -33,3 +33,13 @@ This option can be deactivated to have a permanent Announcement.
 ![Broadcast message display not dismissible](assets/broadcast_message_display_notdismissible.png)
 
 ⚠️ Only one Announcement is displayed at a time. Dismissible Announcement are displayed first, then the latest not dismissible Announcement.
+
+## Analytics
+
+!!! tip "Enterprise edition"
```

Comment:
@guillaumejparis what should be display with this line ?

Reply:
![image](https://github.com/OpenCTI-Platform/docs/assets/4204277/27448d6d-e642-410e-931a-3e1ed730f03d)